src/manage_nsfs/manage_nsfs_cli_errors.js

```diff
@@ -16,18 +18,21 @@ class ManageCLIError extends Error {
     /**
      * @param {ManageCLIErrorSpec} error_spec 
      */
-    constructor({ code, message, http_code }) {
+    constructor({ code, message, http_code, detail, stack }) {
         super(message); // sets this.message
         this.code = code;
         this.http_code = http_code;
+        this.detail = detail;
+        this.stack = stack;
```

Comment:
oh now I understand that you needed the original error stack and not this error stack.
so in js Errors have a `cause` attribute that can be set with new Error(message, { cause: another_error }).
we can pass that optional cause to `super(message, { cause })` in beginning of ctor,
and then to_string() can return `cause: this.cause.stack || this.cause.message`

Reply:
sure, stack vs cause - 
```
cause -

{
  "error": {
    "code": "InternalError",
    "message": "The server encountered an internal error. Please retry the request",
    "cause": "ReferenceError: validate_account_args1 is not defined\n    at get_account_status (noobaa-core/src/cmd/manage_nsfs.js:539:5)\n    at manage_account_operations (noobaa-core/src/cmd/manage_nsfs.js:561:15)\n    at account_management (noobaa-core/src/cmd/manage_nsfs.js:308:11)\n    at async main (noobaa-core/src/cmd/manage_nsfs.js:95:13)"
  }
}

stack -

{
  "error": {
    "code": "InternalError",
    "message": "The server encountered an internal error. Please retry the request",
    "stack": "ReferenceError: validate_account_args1 is not defined\n    at get_account_status (noobaa-core/src/cmd/manage_nsfs.js:539:5)\n    at manage_account_operations (noobaa-core/src/cmd/manage_nsfs.js:561:15)\n    at account_management (noobaa-core/src/cmd/manage_nsfs.js:308:11)\n    at async main (noobaa-core/src/cmd/manage_nsfs.js:95:13)"
  }
}
```